gulpfile.js

```diff
@@ -0,0 +1,86 @@
+/* Copyright 2015 Mozilla Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* jshint node:true */
+/* globals target */
+
+'use strict';
+
+var gulp = require('gulp');
+
+require('./make.js');
+
+gulp.task('default', function() {
+  console.log('Available tasks:');
+  var tasks = Object.keys(gulp.tasks);
+  tasks.sort();
+  tasks.forEach(function (taskName) {
+    console.log('  ' + taskName);
+  });
+});
+
+gulp.task('server', function () {
+  console.log();
+  console.log('### Starting local server');
+
+  var WebServer = require('./test/webserver.js').WebServer;
+  var server = new WebServer();
+  server.port = 8888;
+  server.start();
+});
+
+gulp.task('makefile', function () {
+  var makefileContent = 'help:\n\gulp\n\n';
```

Comment:
`\gulp` must be `\tgulp`, otherwise running `gulp makefile` and then `make server` throws `Makefile:2: *** missing separator`.